tests/test_shell.py

```diff
@@ -129,13 +150,35 @@ def test_index(self):
         doc = json.loads(self.call('python -m signac index'.split()))
         assert 'statepoint' in doc
         assert doc['statepoint'] == {'a': 0}
+        doc = json.loads(self.call('python -m signac project --index'.split()))
+        assert 'statepoint' in doc
+        assert doc['statepoint'] == {'a': 0}
         project.open_job({'a': 0}).document['b'] = 0
         doc = json.loads(self.call('python -m signac index'.split()))
         assert 'statepoint' in doc
         assert doc['statepoint'] == {'a': 0}
         assert 'b' in doc
         assert doc['b'] == 0
 
+    def test_document(self):
+        self.call('python -m signac init my_project'.split())
+        self.call('python -m signac project --access'.split())
```

Comment:
This `signac project --access` command doesn't seem related to testing job documents, so it probably goes somewhere else. The generated `signac_access.py` also appears to use deprecated functionality. @csadorf Do you know if this is supposed to be deprecated overall? I'm not familiar with this part of the codebase, I didn't know this feature for creating "access modules" existed and am not sure whether it still serves a purpose.

Reply:
This is deprecated.

Reply:
I created #303 to address this.